### PR TITLE
Update examples/Interactive Widgets

### PR DESCRIPTION
**Pull request by jdfreder**

### CODE REVIEW

**Comment by jdfreder** [comment]
@minrk - the other set of examples I said I would take a pass over.

File: examples/Interactive Widgets/Custom Widget - Hello World.ipynb [hunk 1]
```diff
@@ -783,7 +785,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.4.2"
+   "version": "3.4.0"
```
Comment (Carreau) [inline examples/Interactive Widgets/Custom Widget - Hello World.ipynb]: I'll just OK, on the things I have read to mark my progress on the diff view.

OK.

File: examples/Interactive Widgets/Exploring Graphs.ipynb [hunk 2]
```diff
@@ -78,748 +78,11 @@
...
-       "ERERJ1DAioiIOIECVkRExAkUsCIiIk6ggBUREXECBayIiIgTKGBFREScQAErIiLiBApYERERJ1DA\n",
-       "ioiIOIECVkRExAkUsCIiIk6ggBUREXECBayIiIgTKGBFREScQAErIiLiBP8HyD8bEel+oXYAAAAA\n",
-       "SUVORK5CYII=\n"
-      ],
-      "text/plain": [
-       "<matplotlib.figure.Figure at 0x1076c6e10>"
-      ]
-     },
-     "metadata": {},
-
```
Comment (Carreau) [inline examples/Interactive Widgets/Exploring Graphs.ipynb]: Do you remove output on purpose ? I though we would leave them for nbviewer ?

OK.

**Comment by Carreau** [comment]
+1, most of the lines were PNG. 

Just the question about not keeping output, that I find weird.

**Comment by jdfreder** [comment]
Well the widgets don't render in nbviewer anyways, so widget output without the widgets that produced it is a little weird in every case except for `interact`.  If you want the outputs, I'd prefer to open a separate PR to this one that runs the notebooks for /Notebook and /Interactive widgets.

**Pull request merged**